pkg/services/accesscontrol/middleware.go

```diff
@@ -59,6 +63,10 @@ func Middleware(ac AccessControl) func(Evaluator) web.Handler {
 }
 
 func authorize(c *contextmodel.ReqContext, ac AccessControl, user identity.Requester, evaluator Evaluator) {
+	ctx, span := tracer.Start(c.Req.Context(), "accesscontrol.authorize")
+	defer span.End()
+	c.Req = c.Req.WithContext(ctx)
+
 	injected, err := evaluator.MutateScopes(c.Req.Context(), scopeInjector(scopeParams{
```

Comment:
I'd also replace the `c.Req.Context()` calls (this and the one below)

```suggestion
	injected, err := evaluator.MutateScopes(ctx, scopeInjector(scopeParams{
```

Reply:
we call `web.Params(c.Req)`  which calls `r.Context().value` so I think we should probably leave this one